tests/filecheck/dialects/arith/arith_ops.mlir

```diff
@@ -205,3 +205,15 @@
 %m_const = arith.constant dense<1.678900e-01> : memref<64xf32>
 // CHECK-NEXT: %t_const = arith.constant dense<1.234500e-01> : tensor<16xf32>
 // CHECK-NEXT: %m_const = arith.constant dense<1.678900e-01> : memref<64xf32>
+
+%fp = arith.constant 2.0 : f32
+%to_si = arith.fptosi %fp : f32 to i32
+%from_si_to_fp = arith.sitofp %to_si : i32 to f32
```

Comment:
These already existed, but were not tested here for some reason, so I added them here